tests/actions/mkloaders_test.py

```diff
@@ -101,4 +102,12 @@ def test_symlink_target_missing(tmp_path):
 
     # Act & Assert
     with pytest.raises(FileNotFoundError):
-        grubimage.symlink(target, link)
+        mkloaders.symlink(target, link)
+
+
+def test_get_syslinux_version():
+    # Arrange & Act
+    result = mkloaders.get_syslinux_version()
```

Comment:
Is this supposed to run a subprocess and take the version from the test environment?

Reply:
Yes in this case since we have `syslinux` available, the real version.

Reply:
Okay, just remember that this test is now interlocked with the test environment, making it harder to (eventually) run unit tests on the dev system.

Reply:
Thanks for pointing out. My lazyness with Docker sometimes is surprising even myself.